uber/templates/regform.html

```diff
@@ -14,7 +14,8 @@
     $(ageGroupSelected);
 
     var staffingClicked = function () {
-        var checked = $.field('staffing') && $.field('staffing').is(':visible:checked');
+        var $staffingCheckbox = $('label[for="staffing"]').find('input[type="checkbox"]');
+        var checked = $staffingCheckbox && $staffingCheckbox.is(':visible:checked');
```

Comment:
Since we now render the staffing checkbox as readonly with a hidden input as the actual field, calling `$.field('staffing')` will not always return the checkbox. If readonly `$.field('staffing')` returns the hidden input. A more general selector is required, `$('label[for="staffing"]').find('input[type="checkbox"]')`.

Although, as I type this, I realize I probably could have used `$('label[for="staffing"] input[type="checkbox"]')`. I can make that change if anyone has a preference.